Library/Homebrew/cask/lib/hbc/cask.rb

```diff
@@ -85,6 +85,23 @@ def installed?
       !versions.empty?
     end
 
+    def latest_installed_version
```

Comment:
`latest_installed_version` is actually just `versions.last`, since they are sorted by timestamp.